alcs-api/src/application/application-decision-meeting/application-decision-meeting.controller.ts

```diff
@@ -31,9 +38,55 @@ export class ApplicationDecisionMeetingController {
   constructor(
     private appDecisionMeetingService: ApplicationDecisionMeetingService,
     private applicationService: ApplicationService,
+    private appDecDocumentService: ApplicationDocumentService,
     @InjectMapper() private mapper: Mapper,
   ) {}
 
+  @Get('/overview/meetings')
+  @UserRoles(...ANY_AUTH_ROLE)
+  async getMeetings(): Promise<Record<string, UpcomingMeetingBoardMapDto[]>> {
+    const upcomingApps =
+      await this.appDecisionMeetingService.getUpcomingMeetings();
+    const allAppIds = upcomingApps.map((a) => a.uuid);
+    const allApps = await this.applicationService.getAll({
+      uuid: Any(allAppIds),
+    });
+    const allFileNumbers = allApps.map((app) => app.fileNumber);
+    const allDocuments = await this.appDecDocumentService.listAll(
+      allFileNumbers,
+      'decisionDocument',
+    );
+    const mappedApps = allApps.map((app) => {
+      const files = allDocuments.filter(
+        (doc) => doc.applicationUuid === app.uuid,
+      );
+      const meetingDate = upcomingApps.find(
+        (meeting) => meeting.uuid === app.uuid,
+      );
+      return {
```

Comment:
maybe DTO with specifying types?

Reply:
Good idea, the types were pretty wrong and not being checked nicely. Added a new DTO and add some more type safety.

Reply:
Done!